### PR TITLE
OCPQE-13910: hypershift public-private regression test

### PR DESCRIPTION
**Pull request by LiangquanLi930**
[OCPQE-13910](https://issues.redhat.com/browse/OCPQE-13910)

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@kuiwang02 @liangxia Could you help to review this PR, thanks!

**Comment by kuiwang02** [comment]
iamok from golang's view

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -708,6 +708,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-public-private
+- as: aws-ipi-ovn-hypershift-public-private-p1-f21
+  cron: 37 0 24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:717]: I'd like to know when will the tag `@critical` added, and when will not.

Comment (YuLi517) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:717]: For OCP release purpose, we will run full function test. No need to add @critical right now. 
For managed-hypershift (online service delivery), we will minimum test scope. @critical tag is needed here. 
In addition, we will optimize our pipeline in future if hypershift is stable on production. then we will consider some of jobs will just run @critical or @high to reduce resource cost.

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:717]: @liangxia ^^

**Pull request merged**